### PR TITLE
add windows store version for vs16

### PR DESCRIPTION
**Pull request by Dargun**

### CODE REVIEW

File: vs-16-2019-win64-store-10-cxx17.cmake [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_init.cmake")
+
+set(CMAKE_SYSTEM_NAME WindowsStore)
+set(CMAKE_SYSTEM_VERSION 10.0)
+
+polly_init(
+    "Visual Studio 16 2019 Win64 / ${CMAKE_SYSTEM_NAME} ${CMAKE_SYSTEM_VERSION} / C++17"
+    "Visual Studio 16 2019"
```
Comment (NeroBurner) [inline vs-16-2019-win64-store-10-cxx17.cmake]: in https://github.com/cpp-pm/polly/blob/master/vs-15-2017-win64-store-10-cxx17.cmake there is the `Win64` extension. Why is that missing here?

Comment (Dargun) [reply vs-16-2019-win64-store-10-cxx17.cmake]: yea you have right have not seen it, will fix it.

**Comment by NeroBurner** [comment]
Have you tested the toolchain locally or have a CI log from GitHub-Actions?

I don't want to merge something untested

**Comment by Dargun** [comment]
i tested that toolchain locally

**Review by NeroBurner — APPROVED** [review]
looks good now

**Comment by NeroBurner** [comment]
@rbsheth can you approve/merge please

**Review by rbsheth — APPROVED** [review]

**Pull request merged**